test/framework/machinepool_helpers.go

```diff
@@ -160,7 +160,7 @@ func UpgradeMachinePoolAndWait(ctx context.Context, input UpgradeMachinePoolAndW
 		if mp.Spec.Template.Spec.InfrastructureRef.Kind == "DockerMachinePool" {
 			version, err := semver.ParseTolerant(input.UpgradeVersion)
 			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("Failed to parse UpgradeVersion %q", input.UpgradeVersion))
-			if version.GTE(semver.MustParse("1.24.0")) && strings.HasSuffix(mp.Spec.Template.Spec.Bootstrap.ConfigRef.Name, "-cgroupfs") {
+			if version.GTE(semver.MustParse("1.25.0")) && strings.HasSuffix(mp.Spec.Template.Spec.Bootstrap.ConfigRef.Name, "-cgroupfs") {
```

Comment:
I'm not sure this should be changed
@sbueringer opinions?

Reply:
Agree